static/js/markdown.js

```diff
@@ -130,9 +130,23 @@ export function apply_markdown(message) {
                 full_name = match[1];
                 user_id = Number.parseInt(match[2], 10);
 
-                if (!helpers.is_valid_full_name_and_user_id(full_name, user_id)) {
-                    user_id = undefined;
-                    full_name = undefined;
+                if (user_id !== undefined) {
+                    if (full_name === undefined) {
+                        // For @**|id** syntax
+                        const valid_ids = helpers.validate_user_ids([user_id], true);
```

Comment:
Rather than extending `people.validate_user_ids` with an ignore flag, `people.is_known_user_id` would be a more straightforward test.

Reply:
Ohh I didn't knew about that, thanks for mentioning it. I updated the wrapper as `helper.is_valid_user_id` which refers to `people.is_known_user_id`. Please have a look.